source/usage-examples/changeStream.txt

```diff
@@ -0,0 +1,35 @@
+=================
+Watch for Changes
+=================
+
+.. default-domain:: mongodb
+
+You can watch for changes in MongoDB, such as changes to a collection,
+database, or deployment, by using the :node-api:`watch()
+</Collection.html#watch>` method. ``watch()`` opens a :manual:`Change
+Stream</changeStreams/>`, that allows applications to watch for changes
+to data and react to those changes. After opening a ``Change Stream``,
+you can process the change events using the
+:node-api:`changeStream.next() </ChangeStream.html#next>` method.
+``watch()`` takes an optional ``pipeline`` parameter that is an array of
+:manual:`stages </reference/operator/aggregation-pipeline/#stages>` to
+filter the change events output. You can stop watching for changes and
+free resources by closing the ``Change Stream`` using the
+:node-api:`changeStream.close() </ChangeStream.html#close>` method.
+
+Create an `Object
+<https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object>`_
+to specify additional options. Set ``fullDocument`` to ``'updateLookup'``
+to receive an event that contains the changes to the document as well as
+the entire altered document.
+
+The ``watch()`` command returns a ``Change Stream`` instance. The
```

Comment:
Isn't this method explained in the initial paragraph? Do you think the detail belongs in the initial paragraph, or that the method details should be omitted from there so that you can explain them in later paragraphs?

Reply:
In the initial paragraph, we explain that it opens a Change Stream instance. So it is similar, this paragraph is here to illustrate the return type. This usage example follows the M.O.R.E format.

Reply:
Ok, I wonder if you can make this distinction more clear to the reader. Perhaps:
"When the watch() command is successful, it returns an instance of the Change Stream that it opened."

Reply:
Now that I see this is a usage example, I am not sure the strict MORE format makes sense since you're describing more than a single method whereas all the prior ones describe a single method. There's mention of all the following in the first paragraph: `watch()`, `next(), and `close()` (Methods). Then, you've listed Options without specifying which ,method they belong to. Then, you list Return types for both `watch()` and `next()`.
Instead, maybe we can divide this into two sections that follows the same MORE format, but one function for each. The first section can talk about opening a Change Stream and have its own Method, Options, Return, Example, and then the second can be about iterating through them and also have it's own Method, Options, Return, and Example sections.